src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -252,8 +246,8 @@ private JoinableTask ExecuteWithinLockAsync(Func<Task> task)
 
             using (_tasksService.LoadedProject())
             {
-                var watchedEvaluationRules = GetWatchedRules(RuleHandlerType.Evaluation);
-                var watchedDesignTimeBuildRules = GetWatchedRules(RuleHandlerType.DesignTimeBuild);
+                var watchedEvaluationRules = _languageServiceHandlerManager.GetWatchedRules(RuleHandlerType.Evaluation);
+                var watchedDesignTimeBuildRules = _languageServiceHandlerManager.GetWatchedRules(RuleHandlerType.DesignTimeBuild);
```

Comment:
Yeah, see the next review - this was a bump in the road, I pulled the renames earlier than needed. IEvaluationHandler becomes evaluation only. There's no longer a IEvaluationHandler that is used for design-time.